azure-devops/azext_devops/dev/boards/_format.py

```diff
@@ -106,6 +106,8 @@ def transform_work_item_query_result_row_output(row):
                     if len(title) > _WORK_ITEM_TITLE_TRUNCATION_LENGTH:
                         title = title[0:_WORK_ITEM_TITLE_TRUNCATION_LENGTH - 3] + '...'
                     table_row[field_reference.name] = title
+                if field_reference.reference_name == 'System.AssignedTo':
```

Comment:
should n't this be elif (else if) check?
if reference_name is System.Title then first title will be stored, then System.AssignedTo check will fail and value of title stored for field_reference.name will be overwritten with field_reference.reference_name.

Reply:
Nice Catch!